rosdep/python.yaml

```diff
@@ -7568,6 +7568,7 @@ python3-shapely:
   osx:
     pip:
       packages: [shapely]
+  rhel: [python3-shapely]
```

Comment:
The pattern we're following right now uses the `python3_pkgversion` macro to get the platform's default python version. In RHEL 8, this is `3`. In RHEL 7, this is `36`.

Since we started using that pattern, things have simplified. The python36-foo packages now provide a virtual python3-foo package in EPEL 7, but there are still a few straggling packages that haven't adopted that pattern. This package follows the correct pattern, so I think we should start deviating from the established `python3_pkgversion` pattern right now.

tl;dr - this RHEL python rule doesn't look like the others, and I think that's OK.